src/common/marlin_server.cpp

```diff
@@ -71,12 +71,12 @@ typedef struct _marlin_server_t {
     uint8_t pqueue_tail;                                     // copy of planner.block_buffer_tail
     uint8_t pqueue;                                          // calculated number of records in planner queue
     uint8_t gqueue;                                          // copy of queue.length - number of commands in gcode queue
+    uint8_t server_device_state;                             // device state stored by server
```

Comment:
device_state is better ??

Reply:
Its just server's variable device_state, named this way to set it apart from marlin_vars_t's device_state